dealroom_firestore_connector/__init__.py

```diff
@@ -455,3 +471,121 @@ def __log_exception(error_code, ref, identifier, was_retried=False):
         error_logger(message, error_code)
     else:
         logging.error(f"[Error code {error_code}] {message} Retrying...")
+
+
+def get_people_doc_refs(
```

Comment:
DO you wanna try to move this to a new file like: people.py? IF you are in urgency ignore this

Reply:
Yes! Indeed, I actually want to do that fall all the functions and structure the connector much better: https://dealroom.atlassian.net/browse/DN-946
But this needs to be done asap.

Reply:
ah okay